deploy/kubernetes/README.md

```diff
@@ -24,6 +24,36 @@ To apply the daemonset to your cluster with example patches applied:
 kustomize build deploy/kubernetes/overlays/examples | kubectl apply -f -
 ```
 
+### cAdvisor with perf support on Kubernetes
+
+Exemplary modification needed to deploy cAdvisor with perf support are provide in [overlays/examples_perf](overlays/examples_perf) directory (modification to daemonset and configmap with perf events configuration).
```

Comment:
Typos:
 `are provided in`
`Examples of modification` https://english.stackexchange.com/questions/78465/example-vs-exemplary